### PR TITLE
Updating instructions in article about QDK version updates

### PR DESCRIPTION
**Pull request by ricardo-espinoza**
Updating instructions in article about QDK version update for Q# projects.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a44b11de30c610339ff0f920997af95eea5a54e5/articles/install-guide/update.md) | 88 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/74e8b342-17dd-40ec-8cd7-b9b77dd794c8_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/update.md [hunk 1]
```diff
@@ -67,30 +67,48 @@ You can now skip ahead to [update your Visual Studio QDK extension](#update-visu
...
-    ```
+        You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
+
+    - Update the version of the all the Microsoft Quantum packages to the most recently released. The version of the packages is specified in the following lines:
+
+        ```xml
+        <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.11.2004.2825" />
+        ```
+        
+        To determine the most recent QDK version, you can review the [release notes](https://docs.microsoft.com/en-us/quantum/relnotes/).
```
Comment (KittyYeungQ) [inline articles/install-guide/update.md:94]:
```suggestion
        To determine the most recent QDK version, you can review the [release notes](https://docs.microsoft.com/quantum/relnotes/).
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/ccc6859c0483713ac6be06b7f1c0c86293998f7a/articles/install-guide/update.md) | 88 | 65 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fc9f994b-b4a9-4704-b96e-bfc6bceab62d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by KittyYeungQ** [comment]
@Bradben @gillenhaalb @geduardo Do you know anything about the warning about admin 'adkinn'?

**Comment by geduardo** [comment]
No idea @KittyYeungQ, he has admin access to the repo. Weird.
image

Do you know what is happening @adkinn?

**Comment by KittyYeungQ** [comment]
> No idea @KittyYeungQ, he has admin access to the repo. Weird.
> image
> 
> Do you know what is happening @adkinn?

We have a couple of PRs having this same warning.

**Comment by geduardo** [comment]
Yeah is happening for every build.

**Pull request merged**